devel-tools/git-move-submodules

```diff
@@ -1,7 +1,9 @@
 #! /usr/bin/env bash
 #
 # Recursively check outs the most recent version of all submodules on a given
-# branch, and commits the updates to the parents.
+# branch, and commits the updates to the parents. If we have local
+# commit pending that's not yet pushed to origin, we amend the update there.
+# Otherwise, we create new ones.
```

Comment:
This comment doesn't really apply anymore, the `do_commit` logic is commented out; can just remove the both the comment and the code.